srcpkgs/pp/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'pp'
 pkgname=pp
-version=1.0.1
+version=1.0.3
 revision=1
 build_style=gnu-makefile
 short_desc="Simple preprocessor"
 maintainer='Michal V <michal@pmsdev.tk>'
 license="ISC"
-homepage="https://adi.tilde.institute/pp/"
-distfiles="https://adi.tilde.institute/${pkgname}/${pkgname}-${version}.tgz"
-checksum=90501faa2952223d3e1e3bdae94fe6e911b271678075fee792ba30cc8e3499e0
+homepage="https://mkws.sh/pp.html"
+distfiles="https://mkws.sh/pp/pp@${version}.tgz"
+checksum=a0040727486362fb47785c8c39dc340e352a8bf87c889f0b528d13642bae8c06
```

Comment:
`8f7309d5a8d193fb60d36affa56599800f8f849037f7115066a6fd15f5aefac2`

Reply:
huh, how did I manage to mess this up. Thanks